trite-osprey/src/App.css

```diff
@@ -0,0 +1,156 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 10s linear;
+  height: 80px;
+  filter: invert(100%);
+}
+
+.App-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.App-intro {
+  font-size: large;
+  padding: 1em;
+}
+h2{
```

Comment:
I love all the name-spacing you were doing up until this point. I would avoid styling native DOM nodes like `h2` unless you really need to or your doing some sort of baseline reset. Give every style a unique classname and then compose those classnames together.